Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: lint test run
```

Comment:
nit: I usually don't bother with `.PHONY` unless there's an actual chance of collision with a real file/dir